src/people.rs

```diff
@@ -64,13 +82,75 @@ pub trait PersonProperty: Copy {
         false
     }
     #[must_use]
+    fn must_be_initialized() -> bool {
```

Comment:
I don't think that this is correct, because the semantics of "must_be_initialized" is that you need to supply an initializer which is of course forbidden for derived properties.

Reply:
How about we rename it.